Package.swift

```diff
@@ -25,6 +28,9 @@ let package = Package(
         .package(url: "https://github.com/Quick/Nimble.git", .upToNextMajor(from: "9.1.0"))
     ],
     targets: [
+        .target(
+            name: "RDeviceIdentifier",
+            dependencies: ["RSDKUtilsMain"]),
```

Comment:
Why does it need to depend on RSDKUtilsMain ?

Reply:
Using its `Data+Extensions.swift`